src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1167,3 +1167,13 @@
     type: command
     short-summary: List pod identity exceptions in a managed Kubernetes cluster
 """
+
+helps['aks egress-endpoints'] = """
+    type: group
+    short-summary: see detail usage in 'az aks egress-endpoints list'.
```

Comment:
What does this mean?

Reply:
changed to `short-summary: Commands to manage egress endpoints in managed Kubernetes cluster.` to get a clear summary.

Reply:
This command group may be able to extend other sub commands later, right? Such as `aks egress-endpoints show`, `aks egress-endpoints create`...
If so, please change it to `Manage egress-endpoints`

Reply:
I don't get you clear, do you mean `short-summary: Manage egress-endpoints`?

Reply:
I read your updated help and I think it's OK~